### PR TITLE
Refactor documentation for native OTLP endpoint

### PR DESCRIPTION
**Pull request by alanwest**

### CODE REVIEW

File: src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start.mdx [hunk 1]
```diff
@@ -78,24 +94,25 @@ The diagram below shows the flow of data with the collector.
...
-   * [.NET](https://github.com/open-telemetry/opentelemetry-dotnet/tree/master/src/OpenTelemetry.Exporter.OpenTelemetryProtocol)
-   * [PHP](https://github.com/open-telemetry/opentelemetry-php/blob/master/examples/AlwaysOnOTLPExample.php)
-   * [Python](https://opentelemetry-python.readthedocs.io/en/stable/exporter/otlp/otlp.html)
-   * [Ruby](https://github.com/open-telemetry/opentelemetry-ruby/tree/master/exporter/otlp)
-   * [Rust](https://crates.io/crates/opentelemetry-otlp)
-   * [Swift](https://github.com/open-telemetry/opentelemetry-swift/tree/master/Sources/Exporters/OpenTelemetryProtocol)
-   * [...Find additional OTLP language support in GitHub](https://github.com/open-telemetry)
+
+<Callout variant="important">
+The New Relic exporter for the collector will be deprecated in the future. If you are running your own collector, you will instead be able to use the [OpenTelemetry Collector's OTLP exporter]() When we release our [native OTLP endpoint (pre-release)](#otlp), th If you were previously using a New Relic language-specific exporter consider using the OTLP exporter for your language and send data directly to New Relic's [native OTLP endpoint (pre-release)](#otlp).
```
Comment (urbiz-nr) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start.mdx:99]: Hi @alanwest - there's a link missing here.

Comment (alanwest) [inline src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start.mdx:99]:
```suggestion
The New Relic exporter for the collector will be deprecated in the future. If you are running your own collector, you will instead be able to use the [OpenTelemetry Collector's OTLP exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlpexporter) When we release our [native OTLP endpoint (pre-release)](#otlp), th If you were previously using a New Relic language-specific exporter consider using the OTLP exporter for your language and send data directly to New Relic's [native OTLP endpoint (pre-release)](#otlp).
```

Comment (alanwest) [reply src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-quick-start.mdx]: @urbiz-nr yep, that's the link, thanks!

**Review by urbiz-nr — APPROVED** [review]
Tried locally, looks good.

**Comment by urbiz-nr** [comment]
@alanwest - thanks for updating this! Merging.

**Pull request merged**